### ISSUE TITLE
Tutorial

### ISSUE THREAD

**Issue by hooverdn** [microsoft/TypeScript#25427]
You need a basic tutorial about converting the usual javascript situation to typescript:  An HTML file includes several javascript files and the later files depend on the earlier ones.  

Would that all God's people had tutorials as good as Ruby on Rails.

**Comment by mhegazy** [comment]
Have you looked at http://www.typescriptlang.org/docs/handbook/migrating-from-javascript.html? if so what are the missing parts of that tutorial?

**Comment by hooverdn** [comment]
Mohamed,
My problems with that article are that

   - It is about Javascript to run in node.js, not in a browser, and:

   - It puts using a bundler up front.  There are a lot of bundlers out
   there and I use rollup, not webpack or gulp, so that makes me say, "Good
   grief, do I have to learn to use *another* bundler just to do this tutorial.

I found "Converting a JavaScript project to Typescript, one file at a time"
<https://dev.to/suhas_chatekar/converting-a-javascript-project-to-typescript-one-file-at-a-time>
to be a more usable tutorial.  It, too starts out with a node project and a
bundler (webpack), but I found that if I just ignored the first half of it,
the second half told me *almost* everything I needed to know to convert a
browser-oriented javascript project to typescript (without using any
bundler).

The main thing it didn't tell me is how to get the compiler, tsc, to emit
es5 code that doesn't contain import statements or export decorators.  (I
have been just deleting them maually.)

I could go into more detail about what a browser-oriented typescript
tutorial should cover if anybody cares to hear about it.

Thanks,

Doug Hoover
hooverdn@gmail.com
